timemachine/fe/rbfe.py

```diff
@@ -408,9 +408,10 @@ def estimate_free_energy_given_initial_states(initial_states, protocol, temperat
     ukln_by_lambda_by_component = np.array(ukln_by_component_by_lambda).swapaxes(0, 1)
 
     def pair_overlap(u_kln):
```

Comment:
Might be useful to move out of this function and add a basic test.

Reply:
Done in [986dc1e](https://github.com/proteneer/timemachine/pull/823/commits/986dc1e3cd964901dcc7538b3a91d6d12545094d). Unfortunately, I wasn't able to figure out a simple example where the pair BAR overlap computed on `u_kln` is different from that on `u_kln.swapaxes(0, 1)` (i.e. the test added also passes with the bug present). I still want to understand why the ordering of axes doesn't seem to matter for gaussians, but maybe this is good enough to merge for now?